framework/src/org/checkerframework/common/value/util/Range.java

```diff
@@ -20,6 +20,9 @@
     /** The upper bound of the interval, inclusive. */
     public final long to;
 
+    /** Should this range take overflow into account or ignore it? */
```

Comment:
The documentation needed to be changed. We originally had this as an instance variable, and decided to switch to a static variable because it makes the implementation cleaner.